config/jupyterhub_config.py

```diff
@@ -0,0 +1,26 @@
+"""
+This is the JupyterHub configuration file. It is used to configure the JupyterHub server.
+Refer to the JupyterHub documentation for more information:
+https://jupyterhub.readthedocs.io/en/latest/tutorial/getting-started/config-basics.html
+"""
+import os
+
+from jupyterhub_config.custom_spawner import VirtualEnvSpawner
+
+c = get_config()
+
+# Set the authenticator class
+# TODO: Change the authenticator class to a secure one (e.g. GitHubOAuthenticator)
+c.JupyterHub.authenticator_class = 'jupyterhub.auth.DummyAuthenticator'
+c.Authenticator.allowed_users = {'spark_user', 'test_user1', 'test_user2'}
+c.DummyAuthenticator.password = os.environ['JUPYTERHUB_ADMIN_PASSWORD']
```

Comment:
is there an authenticator that allows setting a password per user?

Reply:
I think there is but I want to explore that in the future.

Reply:
:+1: